src/frontend/note.js

```diff
@@ -174,6 +174,16 @@ class Note {
         this.#editor_element.classList.add('hidden');
     }
 
+    filter(value) {
+        if ((this.#name.includes(value)) || (
+            this.#editor.value().includes(value))) {
```

Comment:
```suggestion
        if ((this.#name.toLowerCase().includes(value.toLowerCase())) || (
            this.#editor.value().toLowerCase().includes(value.toLowerCase()))) {
```
searching case insensitive is much more convenient